lib/avo.rb

```diff
@@ -18,6 +18,7 @@ module Avo
   IN_DEVELOPMENT = ENV["AVO_IN_DEVELOPMENT"] == "1"
   PACKED = !IN_DEVELOPMENT
   COOKIES_KEY = "avo"
+  ACTIONS_TURBO_FRAME_ID = :actions_show
```

Comment:
Great! We should use this on our docs on the multiple action flux https://docs.avohq.io/3.0/actions.html#:~:text=you%20may%20want%20to%20redirect%20to%20another%20action.%20